erpnext/stock/doctype/item/item.json

```diff
@@ -1122,7 +1122,7 @@
  "idx": 2,
  "image_field": "image",
  "max_attachments": 1,
- "modified": "2021-02-02 00:19:02.360290",
+ "modified": "2021-04-20 07:23:16.258679",
```

Comment:
why?

Reply:
There were some changes I made that I erased later, thats probably why its showing an updated modified date